crates/valence_command/src/manager.rs

```diff
@@ -151,10 +151,11 @@ fn update_client_command_tree(
         let mut new_root = None;
 
         while let Some((parent, node)) = to_visit.pop() {
-            if already_visited.contains(&(parent.map(|(_, edge)| edge), node)) {
+            if already_visited.contains(&(parent.map(|(node_id, _)| node_id), node)) {
                 continue;
             }
-            already_visited.insert((parent.map(|(_, edge)| edge), node));
+            already_visited.insert((parent.map(|(node_id, _)| node_id), node));
+
```

Comment:
This change seems unrelated to the upgrade.

Reply:
I will make a new pr for this